packages/storybook/src/nimble/breadcrumb/breadcrumb.stories.ts

```diff
@@ -46,8 +46,15 @@ const itemHrefDescription = `${defaultHrefDescription} If the last breadcrumb it
 export const _standardBreadcrumb: StoryObj<BreadcrumbArgs> = {
     // prettier-ignore
     render: createUserSelectedThemeStory(html`
+    <div
```

Comment:
todo, update stories options to be like tabs (instead of json config options like many items, wide items, etc)